packages/message-input/src/vaadin-message-input-mixin.js

```diff
@@ -116,15 +116,6 @@ export const MessageInputMixin = (superClass) =>
             }
           });
 
-          // With Lit version, input element renders asynchronously and it will
-          // override the `rows` attribute set to `1` in the `minRows` observer.
-          // Workaround: perform update twice to run the observer synchronously.
```

Comment:
The second `performUpdate()` is still needed (can be checked by running snapshot tests for Lit version).

Reply:
Good catch. Fixed this by making the `inputElement` property sync.